gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py

```diff
@@ -0,0 +1,457 @@
+
+    @log_helpers.log_method_call
+    def delete_healthmonitor(self, context, healthmonitor):
+        loadbalancer = healthmonitor['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._delete(
+            context, healthmonitor['tenant_id'],
+            'healthmonitor', nf, healthmonitor=healthmonitor)
+
+    # TODO(jiahao): L7policy support not implemented
+    # disable L7policy
+    # def create_l7policy(self, context, l7policy):
+    #     self._post(
+    #         context, l7policy['tenant_id'],
+    #         'l7policy', l7policy=l7policy)
+    #
+    # def delete_l7policy(self, context, l7policy):
+    #     self._delete(
+    #         context, l7policy['tenant_id'],
+    #         'l7policy', l7policy=l7policy)
+    #
+    # def create_l7policy_rule(self, context, rule, l7policy_id):
+    #     self._post(
+    #         context, rule['tenant_id'],
+    #         'rule', rule=rule)
+    #
+    # def delete_l7policy_rule(self, context, rule):
+    #     self._delete(
+    #         context, rule['tenant_id'],
+    #         'rule', rule=rule)
+    #
+    # def _get_lb_context(self, context, filters):
+    #     args = {'context': context, 'filters': filters}
+    #     db_data = super(Lbv2Agent, self)
+    #     return {'loadbalancers': db_data.get_loadbalancers(**args),
+    #             'listeners': db_data.get_listeners(**args),
+    #             'pools': db_data.get_pools(**args),
+    #             'pool_members': db_data.get_pool_members(**args),
+    #             'healthmonitors': db_data.get_healthmonitors(**args),
+    #             'l7policies': db_data.get_l7policies(**args),
+    #             'l7policy_rules': db_data.get_l7policy_rules(**args)}
+
+
+class LoadbalancerV2Notifier(object):
+
+    def __init__(self, conf, sc):
+        self._sc = sc
+        self._conf = conf
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def _prepare_request_data(self, context, nf_id,
+                              resource_id, lb_id,
+                              service_type):
+        request_data = None
+        try:
+            request_data = common.get_network_function_map(
+                context, nf_id)
+            # Adding Service Type #
+            # TODO(jiahao): is the key name "loadbalancer_id" correct?
+            request_data.update({"service_type": service_type,
+                                 "loadbalancer_id": lb_id,
+                                 "neutron_resource_id": resource_id,
+                                 "LogMetaID": nf_id})
+        except Exception as e:
+            LOG(LOGGER, 'ERROR', '%s' % (e))
+            return request_data
+        return request_data
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def _trigger_service_event(self, context, event_type, event_id,
```

Comment:
Used for sending event to visibility when vip deleted. Identify corresponding resource in LBaaSV2 and invoke this.